modules/data.atmosphere/R/downscale_solar_geom.R

```diff
@@ -0,0 +1,34 @@
+##' @name downscale_solar_geom
+##' @title solar_geom
+##' @export
+##' @author Mike Dietze
+##' @description calculates potential top-of-atmosphere shortwave radiation as a function of day of year and location
+##' @param doy time as day of year. Integers indicated midnight.
+##' @param lon longitude
+##' @param lat latitude
+downscale_solar_geom <- function(doy, lon, lat) {
```

Comment:
This looks like a >90% duplicate of the functions in https://github.com/PecanProject/pecan/blob/develop/modules/data.atmosphere/R/solar_angle.R. I would move this function into that same file, refactor any duplicate pieces (for example, I bet some of this can use the `equation_of_time` function), and, if there are key differences, emphasize them in the function documentation.